app/views/content/life-as-a-teacher/explore-subjects/modern-languages/_article.html.erb

```diff
@@ -0,0 +1,142 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach modern languages</h2>
+    <p>As a language teacher, you’ll teach the linguistic aspects of language and introduce pupils to the cultural contexts that give language its richness and depth.</p>
+      <p>In addition to the personal rewards of teaching modern languages, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">generous and secure pension</a></li>
+        <li>more holiday days than people in many other professions</li>
+        <li>opportunities for progression</li>
+      </ul>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "There’s a lot of sharing of stories and the kids can be so funny, they really crack me up. Plus, I enjoy the holidays. I get to travel still and can keep up with my languages.",
+    name: "Jacqui, languages teacher",
+    large: true
+  ) %>
+  </section>
+
+<section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Learning a new language will give pupils cultural awareness and enhanced communication skills that can lead to exciting careers.</p>
+      <p>Popular careers range from flight attendants and ski instructors to interpreters and tour guides. Pupils may also have advantages in careers such as journalism, public relations or finance.</p>
```

Comment:
Wonder if the order of this needs rejigging to have interpreters and tour guides first?